cli/bpmetadata/proto/bpmetadata.proto

```diff
@@ -453,6 +468,43 @@ message BlueprintVariable {
   string var_type = 3; // @gotags: json:"varType,omitempty" yaml:"varType,omitempty"
   google.protobuf.Value default_value = 4; // @gotags: json:"defaultValue,omitempty" yaml:"defaultValue,omitempty"
   bool required = 5; // @gotags: json:"required,omitempty" yaml:"required,omitempty"
+  // Incoming connections to this variable.
```

Comment:
Can we define "connections" a bit more explicitly so it's easier for a someone without ADC context to understand?

Reply:
good point, added some more details